ts/Series/BoxPlot/BoxPlotSeries.ts

```diff
@@ -296,26 +356,25 @@ class BoxPlotSeries extends ColumnSeries {
                 }
 
                 // The whiskers
-                if (whiskerLength) {
-                    const whiskerStrokeWidth = point.whiskers.strokeWidth();
-                    highPlot = crisp(point.highPlot, whiskerStrokeWidth);
-                    lowPlot = crisp(point.lowPlot, whiskerStrokeWidth);
-                    pointWiskerLength = (
-                        typeof whiskerLength === 'string' &&
-                        (/%$/).test(whiskerLength)
-                    ) ?
-                        halfWidth * parseFloat(whiskerLength) / 100 :
-                        Number(whiskerLength) / 2;
-                    d = [
-                        // High whisker
-                        ['M', crisp(stemX - pointWiskerLength), highPlot],
-                        ['L', crisp(stemX + pointWiskerLength), highPlot],
+                if (pointWhiskerLength) {
+                    const halfWidth = width / 2,
+                        whiskers = this.getWhiskerPair(
+                            halfWidth,
+                            stemX,
+                            (
+                                point.upperWhiskerLength ||
+                                options.upperWhiskerLength ||
```

Comment:
I think we should use `??` rather than `||` here. It is intuitive for an implementer to disable the whisker by setting the length to 0. See https://jsfiddle.net/highcharts/xw25ubn3/2/ . This is also how a `whiskerLength` of 0 behaves.